package/MDAnalysis/coordinates/base.py

```diff
@@ -978,13 +981,16 @@ def __repr__(self):
             natoms=self.n_atoms
         ))
 
-    def add_auxiliary(self, auxname, auxdata, format=None, **kwargs):
+    def add_auxiliary(self, auxname: Union[str, List[str]],
+                      auxdata: Union[str, AuxReader],
+                      auxterm: Union[str, List[str], None] = None,
```

Comment:
For types that have `Union[X,Y,None]` you should probably use `Optional[X,Y]` instead, see https://docs.python.org/3/library/typing.html#typing.Optional